packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,195 @@
+});
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+// use the stamper in the client config
+const browserConfig: TurnkeySDKClientConfig = {
+  stamper: stamper,
+  apiBaseUrl: "https://api.turnkey.com",
+  organizationId: <ORGANIZATION_ID>,
+};
+
+// create a TurnkeyClient with the initialized Telegram Cloud Storage Stamper
+const client = new TurnkeyBrowserClient(browserConfig);
+
+// make a request with the client
+const whoamiResponse = await client.getWhoami({
+  organizationId: <ORGANIZATION_ID>,
+});
+```
+
+Use an existing key that has been previously stored in Telegram Cloud Storage at the default API key location key location
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+import { TurnkeyBrowserClient, TurnkeySDKClientConfig } from "@turnkey/sdk-browser";
+
+// create a new Telegram Cloud Storage stamper
+const stamper = await TelegramCloudStorageStamper.create();
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+// use the stamper in the client config
+const browserConfig: TurnkeySDKClientConfig = {
+  stamper: stamper,
+  apiBaseUrl: "https://api.turnkey.com",
+  organizationId: <ORGANIZATION_ID>,
+};
+
+// create a TurnkeyClient with the initialized Telegram Cloud Storage Stamper
+const client = new TurnkeyBrowserClient(browserConfig);
+
+// make a request with the client
+const whoamiResponse = await client.getWhoami({
+  organizationId: <ORGANIZATION_ID>,
+});
+```
+
+View an entry in Telegram Cloud Storage without inserting an API key, note the usage difference between the `new` and `.create()` here. `.create()` will do the work of getting/setting an API key in Cloud Storage where as  `new` will not
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = new TelegramCloudStorageStamper();
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+// the key used to index Telegram Cloud Storage
+const telegramCloudStorageKey = "telegramCloudStorageKey";
+
+// get the item stored in Telegram Cloud Storage returned as a string
+const item = await stamper.getItem(telegramCloudStorageKey);
+
+if(!item) {
+  // failed retrieving item
+}
+
+```
+
+Insert a new API key into Cloud Storage at a specified key. This is just storing an API key, without using `.setSigningKey()` the key will not be used for signing. 
+
+```ts
+import { TelegramCloudStorageStamper } from "@turnkey/telegram-cloud-storage-stamper";
+
+// create a new Telegram Cloud Storage Stamper
+const stamper = new TelegramCloudStorageStamper();
+
+if (!stamper) {
+  // There was a problem initializing the stamper
+}
+
+const apiPublicKey = "...";
+const apiPrivateKey = "...";
+
+// the key used to index Telegram Cloud Storage
+const telegramCloudStorageKey = "telegramCloudStorageKey";
+
+// insert the API key in Telegram Cloud Storage
```

Comment:
for the email auth type use case I imagine?

Reply:
Can be anything the user desires here really :) Just tried to be flexible for whatever the user might need. Certainly email auth is a good example